.dockerignore

```diff
@@ -0,0 +1,74 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# LLVM IR files
+*.ll
+*.ll-*
+
+# Folders
+_obj
+_test
+tests/cilium-files
+test/test_results
+test/*.json
+_build/
+
+# Architecture specific extensions/prefixes
+*.cgo1.go
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+*.exe
+*.test
+*.prof
+
+*.swn
+*.swp
+.vagrant
+vagrant.kubeconfig
+coverage.out
+coverage-all.out
+coverage-all.html
+
+.DS_Store
+.idea/
+*.plist
+
+cilium/bash_autocomplete
+*.swo
+outgoing
+
+*cscope.files
+*cscope.out
+*cscope.in.out
+*cscope.po.out
+*tags
+
+man/
+
+test/tmp.yaml
+test/*_service_manifest.json
+test/*_manifest.yaml
+test/*_policy.json
+test/*.xml
+
+# Emacs backup files
+*~
+
+# Automatically generated when needed
+.dockerignore
+
+# Temporary files that allow build containers/VMs work without git
+
+test/bpf/_results
+
+# generated from make targets
+*.ok
+.*
+Documentation
```

Comment:
Why is Documentation excluded?

Reply:
To save space in the container image?

```
breo.acalustra.com ➜  cilium git:(FQDNResponseLen) ✗ ls Documentation/_build/
doctrees  spelling
breo.acalustra.com ➜  cilium git:(FQDNResponseLen) ✗ du -chs Documentation
21M     Documentation
21M     total
breo.acalustra.com ➜  cilium git:(FQDNResponseLen) ✗
```

Reply:
Which container image?

Reply:
made the change, Documentation is not included.